modin/pandas/base.py

```diff
@@ -693,9 +743,7 @@ def count(self, axis=0, level=None, numeric_only=False):
                 # error thrown by pandas
                 raise TypeError("Can only count levels on hierarchical columns.")
 
-            if isinstance(level, str):
-                level = self.axes[axis].names.index(level)
-            return self.groupby(level=level, axis=axis).count()
+            return self._handle_level_agg(axis, level, "count")

```

Comment:
Please add the keyword arguments here as well.

Reply:
Made the changes, thanks for the review!